govgen/chain.json

```diff
@@ -0,0 +1,226 @@
+{
+	"$schema": "../chain.schema.json",
+	"chain_name": "govgen",
+	"chain_id": "govgen-1",
+	"website": "https://govgen.io/",
+	"pretty_name": "GovGen",
+	"status": "live",
+	"network_type": "mainnet",
+	"bech32_prefix": "govgen",
```

Comment:
Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
done here https://github.com/satoshilabs/slips/pull/1687